### PR TITLE
Implement CI/CD pipelines using GitHub Actions to auto update data in this repo.

### PR DESCRIPTION
**Pull request by geoffrey1330**

### CODE REVIEW

**Review by anuveyatsu — CHANGES_REQUESTED** [review]
Hi @geoffrey1330 thanks for the PR. Please, note this only calls `make data` script which fetches data but it doesn't push it back to GitHub. Our goal is obviously to keep data updated in the repository.

**Comment by anuveyatsu** [comment]
Also note that I have renamed the title of this PR to make it more descriptive.

**Comment by geoffrey1330** [comment]
@anuveyatsu i understand fully the goal now. i will make all necessary changes. 
quick question, the push of data to github should be done on my branch?

**Comment by anuveyatsu** [comment]
@geoffrey1330 I think it should only push on the master branch to make things easier.

**Comment by geoffrey1330** [comment]
@anuveyatsu  Alright i'm on it now

**Comment by geoffrey1330** [comment]
@anuveyatsu here is the new commit that commits and push updated data to the repo 
https://github.com/datasets/ppp/pull/12/commits/d5e20c92fe87179160abbf7212d18e166d563643

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+name: ppp github action ci 
+
+on: 
+  push:
+    branches:
+      - master
+  pull_request:
```
Comment (anuveyatsu) [inline .github/workflows/ci.yaml:7]: @geoffrey1330 since we're going to push updates to master branch, this job shouldn't run on pull requests.

Comment (geoffrey1330) [reply .github/workflows/ci.yaml:7]: @anuveyatsu Alright i will remove the pull request trigger

Comment (geoffrey1330) [reply .github/workflows/ci.yaml]: @anuveyatsu here is the new commit 
https://github.com/datasets/ppp/pull/12/commits/2d7456d4a71fb947e316e2677a1e9e8981d890b3

**Pull request merged**